indexing-service/src/test/java/io/druid/indexing/common/TestRealtimeTask.java

```diff
@@ -53,7 +54,7 @@ public TestRealtimeTask(
         id,
         taskResource,
         new FireDepartment(
-            new DataSchema(dataSource, null, new AggregatorFactory[]{}, null), new RealtimeIOConfig(
+            new DataSchema(dataSource, null, new AggregatorFactory[]{}, null, new DefaultObjectMapper()), new RealtimeIOConfig(
```

Comment:
Can the object mapper be a class field simply to fit in with other test convention?